src/SDKs/DataBox/DataBox.Tests/DataBox.Tests.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>DataBox.Tests</PackageId>
+    <Description>DataBox.Tests class library</Description>
+    <AssemblyName>DataBox.Tests</AssemblyName>
+    <VersionPrefix>1.0.0</VersionPrefix>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.Compute" Version="18.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="10.1.0-preview" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Management.DataBox\Microsoft.Azure.Management.DataBox.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Update="SessionRecords\**\*.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+  </ItemGroup>
+
+  <!--Do not remove until VS Test Tools fixes #472-->
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.6.2" />
+    <PackageReference Update="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.12" />
+    <PackageReference Update="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="1.7.2" />
+  </ItemGroup>
+  <ItemGroup>
+    <Folder Include="SessionRecords\" />
```

Comment:
@kushalvms this is not needed. Removing it from csproj will still add the required directory to your projects

Reply:
removed